tests/inputFields.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { test, expect } from '@playwright/test';
+//vreate a before each to go to the url, click on the navigation bar PET TYPES and assertion for "Pet Types" text
+test.beforeEach(async({page}) => {
+    await page.goto('baseURL')
+ await page.getByText('Pet Types').click()
+
+ await expect(page.locator('h2')).toHaveText("Pet Types")
+})
+
+//3. Click on "Edit" button for the "cat" pet type
+test('change the name of the cat', async({page}) => {
+    const petTypesCat= page.locator('button').filter({hasText:"Edit"}).first()
+    await petTypesCat.click()
+
+    /*
+    Add assertion of the "Edit Pet Type" text displayed*/
+
+    await expect (page.locator('h2')).toHaveText('Edit Pet Type')
+
+
+//Change the pet type name from "cat" to "rabbit" and click "Update" button
+const correctPetType = page.locator('#name')
```

Comment:
Use a better naming.
This is not a `correctPetType`.
This locator is `editPetTypeInputField` or `editPetNameInputField`
See the difference? :)